test/functional/autocmd/cmdline_spec.lua

```diff
@@ -78,56 +78,56 @@ describe('cmdline autocommands', function()
     ]])
 
     feed("put ='lorem ipsum'<cr>")
-    screen:expect([[
-                                                                              |
+    screen:expect{grid=[[
       {4:                                                                        }|
       :                                                                       |
       {2:E5500: autocmd has thrown an exception: Vim(echoerr):FAIL}               |
       :put ='lorem ipsum'                                                     |
-      {2:E5500: autocmd has thrown an exception: Vim(echoerr):very error}         |
+      {2:E5500: autocmd has thrown an exception: %s}                              |
```

Comment:
literal `%s` is not wanted. with `semsg` this test should need less changes.

Reply:
Doh, now it makes sense. Thanks.